t/perlcriticrc

```diff
@@ -1,41 +1,72 @@
 
-# $Id$
-
 severity = 5
-verbose  = 8
+verbose  = 3
 
 #------------------------------------------------------------------------------
 
+# REF: https://metacpan.org/pod/Perl::Critic::Policy::Documentation::RequirePodSections
 [Documentation::RequirePodSections]
 lib_sections    = NAME|DESCRIPTION|SYNOPSIS|AUTHORS|COPYRIGHT
 script_sections = NAME|DESCRIPTION|SYNOPSIS|AUTHORS|COPYRIGHT
 
+# REF: https://metacpan.org/pod/Perl::Critic::Policy::TestingAndDebugging::RequireUseWarnings
 [TestingAndDebugging::RequireUseWarnings]
+
+# REF: https://metacpan.org/pod/Perl::Critic::Policy::ValuesAndExpressions::ProhibitMagicNumbers
 [ValuesAndExpressions::ProhibitMagicNumbers]
 
-#Might be to late to address
-[-Subroutines::ProhibitExplicitReturnUndef]
+# Might be to late to address, so it is disabled at various places in the code, could perhaps be 
+# revisited with a major release, since it would break backwards compatibility, the recommeded 
+# solution is bare return
+# REF: https://metacpan.org/pod/Perl::Critic::Policy::Subroutines::ProhibitExplicitReturnUndef
+[Subroutines::ProhibitExplicitReturnUndef]
 
-#The violater of this is an overwrite for Class::Accessor
+# The violater of this is an overwrite for Class::Accessor
+# REF: https://metacpan.org/pod/Perl::Critic::Policy::NamingConventions::ProhibitAmbiguousNames
 [-NamingConventions::ProhibitAmbiguousNames]
 
-#We are using Constants
+# We are using Constants
+# Perhaps exchange this for Readonly?
```

Comment:
The code actually introduces the use of `Readonly`. Shouldn't this question be removed and changed to a confirmation "Instead, we use Readonly"?